cassandra/policies.py

```
@@ -493,40 +730,168 @@ def check_supported(self):
+                    if cluster_metadata._tablets:
+                        tablet = cluster_metadata._tablets.get_tablet_for_key(
+                            keyspace, query.table, token)
+                        if tablet is not None:
+                            replicas_mapped = {r[0] for r in tablet.replicas}
+                            child_plan = child.make_query_plan(keyspace, query)
+                            replicas = [host for host in child_plan if host.host_id in replicas_mapped]
+
+                    if not replicas:
+                        try:
+                            replicas = token_map.get_replicas(keyspace, token)
+                        except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)